enqueue/enqueue-bundle/0.3/etc/packages/test/enqueue.yaml

```diff
@@ -0,0 +1,6 @@
+enqueue:
+    transport:
+        default: 'null'
+        'null': ~
```

Comment:
@javiereguiluz In this context 'null' is the name of the transport. The transport does not send nor receive anything. It is there just to mock a messaging API and useful in tests for example. 

I've not come up with a better name for the transport.